front/src/components/App/Help/Help.tsx

```diff
@@ -0,0 +1,73 @@
+import { useState } from "react";
+import { Switch, Route, BrowserRouter as Router } from "react-router-dom";
+import { v4 as uuid_v4 } from "uuid";
+import Button from "../../UI/Button/Button";
+import Header from "../../UI/Header/Header";
+import RouteLink from "../../UI/RouteLink/RouteLink.styled";
+import EditTabs from "../../UI/Tabs/EditTabs/EditTabs";
+import Tabs from "../../UI/Tabs/Tabs";
+import { APP_PAGE } from "../App";
+import { HelpStyled } from "./Help.styled";
+
+
+
+/* ONLY FOR DEVELOPING PROCESS */
+const sampleElements = [
+    { id: uuid_v4(), name: "Allegro", content: "In up so discovery my middleton eagerness dejection explained. Estimating excellence ye contrasted insensible as. Oh up unsatiable advantages decisively as at interested. Present suppose in esteems in demesne colonel it to. End horrible she landlord screened stanhill. Repeated offended you opinions off dissuade ask packages screened. She alteration everything sympathize impossible his get compliment. Collected few extremity suffering met had sportsman." },
+    { id: uuid_v4(), name: "Amazon", content: "Amazon is cool"},
+];
+
+/* END OF DEVELOPER SECTION */
+
+export interface PackageData {
+    id: string;
+    name: string;
+    content: string;
+};
+
+const Help = () => {
+    //States
+    const [elements, setElements] = useState<PackageData[]>(sampleElements);
+
+    //parameter just for testing
+    const addElementHandler = () => {
+
+        const request = {
+            id: uuid_v4(),
+            name: "New Element",
+            content: "New Content",
+        }
+        setElements([...elements, request]);
+    };
+
+    const updateElementHandler = (element: PackageData) => {
```

Comment:
sama ta funkcja nie jest logiczną częśćią Help.tsx. Powinna ona być zadeklarowana w komponencie z Tabsami bo to na nich ona operuje.